client/components/phone-input/index.jsx

```diff
@@ -24,6 +24,7 @@ const PhoneInput = React.createClass( {
 	},
 
 	getCountry( countryCode = this.props.countryCode ) {
+		countryCode = countryCode.toLowerCase();
```

Comment:
I've switched everything to upper case, but since I have a other PR's on air, I put some `toUpperCase()` functions to `props.countryCode` to prevent creating PR dependencies. So it doesn't look nicer at this point, but we have a clean and simple migration path once everything gets merged in.